Permissioned DEX XLS.md

```diff
@@ -0,0 +1,479 @@
+|`DomainID`| |`string`|`Hash256`|The domain to delete.|
+
+### 4.2. Failure Conditions
+
+* The domain specified in `DomainID` doesn't exist.
+* The account isn't the owner of the domain.
+
+### 4.3. State Changes
+
+* Deletes a `PermissionedDomain` object.
+
+## 5. On-Ledger Object: `Offer`
+
+The `Offer` object tracks an offer placed on the CLOB DEX. This object type already exists on the XRPL, but is being extended as a part of this spec to also support permissioned DEX domains.
+
+### 5.1. Fields
+
+<details>
+<summary>
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/offer) are the existing fields for the `Offer` object.
+</summary>
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Account`|✔️|`string`|`AccountID`|The address of the account that owns this offer.|
+|`BookDirectory`|✔️|`string`|`Hash256`|The ID of the offer directory that links to this offer.|
+|`BookNode`|✔️|`string`|`UInt64`|A hint indicating which page of the offer directory links to this entry, in case the directory consists of multiple pages.|
+|`Expiration`| |`number`|`UInt32`|Indicates the time after which this offer is considered unfunded.|
+|`LedgerEntryType`|✔️|`string`|`UInt16`|The value `0x006F`, mapped to the string `Offer`, indicates that this is an offer entry.|
+|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the owner directory links to this entry, in case the directory consists of multiple pages.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this entry.|
+|`PreviousTxnLgrSeq`|✔️|`number`|`UInt32`|The ledger index that contains the transaction that most recently modified this object.|
+|`Sequence`|✔️|`number`|`UInt32`|The `Sequence` value of the `OfferCreate` transaction that created this offer. Used in combination with the `Account` to identify this offer.|
+|`TakerPays`|✔️|`object`|`Amount`|The remaining amount and type of currency requested by the offer creator.|
+|`TakerGets`|✔️|`object`|`Amount`|The remaining amount and type of currency being provided by the offer creator.|
+</details>
+
+We propose these additions:
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`DomainID`| |`string`|`Hash256`|The domain that the offer must be a part of.|
+
+#### 5.1.1. `DomainID`
+
+A permissioned offer has a `DomainID` field included.
+
+An open offer does not need to include a `DomainID` field.
+
+### 5.2. Offer Invalidity
+
+An offer on the orderbook can become invalid if the domain specified by the `DomainID` field is deleted, and will no longer be filled. It will be automatically deleted when crossed, like an unfunded offer.
+
+The same will be true if the credential that allows the offer's owner to be a member of the domain expires or is deleted.
```

Comment:
Would it be invalid to submit an OfferCreate with a DomainID where the AcceptedTokens field of the PermissionedDomain don't overlap with any or all of the currencies specified TakerPays and/or TakerGets of the OfferCreate?

Reply:
Yes. That would make it an invalid domain offer. The tokens in the offer must be a subset of the ones in the `AcceptedTokens`.